lib/SPIRV/OCLToSPIRV.cpp

```diff
@@ -168,9 +168,11 @@ bool OCLToSPIRVBase::runOCLToSPIRV(Module &Module) {
   auto Src = getSPIRVSource(&Module);
   // This is a pre-processing pass, which transform LLVM IR module to a more
   // suitable form for the SPIR-V translation: it is specifically designed to
-  // handle OpenCL C built-in functions and shouldn't be launched for other
-  // source languages
-  if (std::get<0>(Src) != spv::SourceLanguageOpenCL_C)
+  // handle OpenCL C/CPP and CPP for OpenCL modules and shouldn't be launched
```

Comment:
In comments we can refer to the official language names:

```suggestion
  // handle OpenCL C/C++ and C++ for OpenCL modules and shouldn't be launched
```

Reply:
Done in commit
Thanks